Tabloid/client/src/components/Comments/CommentForm.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { FormGroup, Input, Label, Button, Form } from "reactstrap";
+import { addComment } from "../../modules/commentManager";
+import { useParams } from "react-router-dom";
+
+const CommentForm = () => {
+  const history = useHistory();
+  const { postId } = useParams();
+  const emptyComment = {
+    PostId: 1,
+    Subject: "",
+    Content: ""
+  };
+
+  const [comment, setComment] = useState(emptyComment);
+
+  const handleInputChange = (evt) => {
+    const value = evt.target.value;
+    const key = evt.target.id;
+
+    const commentCopy = { ...comment };
+
+    commentCopy[key] = value;
+    setComment(commentCopy);
+  };
+  const handleSave = (evt) => {
+    evt.preventDefault();
+    const commentCopy = { ...comment };
+    commentCopy.PostId = parseInt(postId);
+    addComment(commentCopy).then(() => {
+      history.push("/");
```

Comment:
history.push should send the user back to the post detail page for the commented post

Reply:
I have addressed this issue in my new push